src/components/widgets/Map.vue

```diff
@@ -81,7 +81,11 @@
     absolute
     bottom
     color="white"
+    class="flex mt-12"
   />
+  <p v-if="fetchingMission" class="fixed top-[58px] left-[7px] flex text-md font-bold text-white z-30 drop-shadow-md">
```

Comment:
When the map widget is not full-screened, this bar gets on a weird place:

<img width="1090" alt="image" src="https://github.com/user-attachments/assets/8bc68f0c-0b86-4567-a7d8-beed8084a5bb">

You can reuse the logic of the map bottom buttons so its reactive with the map position.

Reply:
Good catch
Lets make this position dynamic